.estafette.yaml

```diff
@@ -177,7 +177,16 @@ releases:
             GKE_DEFAULT_REPOSITORY_5: estafette.secret(XSu_Wd5OQYoSbTE0.t9ihASlW5cKyXfw9QWVu3XnZtI7tceMBqVrqjeyqUIzVAgVf)
-            GKE_DEFAULT_SIDECAR_IMAGE_5: estafette.secret(DF-n_xDqo-0R9K28.0sB6LfLnyihjZz24E58plq35JI8vm8D3c30bXee-DAQvoa71d-21dLes5QSzRen4)
+            GKE_DEFAULT_SIDECAR_IMAGE_5: estafette.secret(DF-n_xDqo-0R9K28.0sB6LfLnyihjZz24E58plq35JI8vm8D3c30bXee-DAQvoa71d-21dLes5QSzRen4)            
+            GKE_CREDENTIAL_6: estafette.secret(0DXFRox5jqKlm9oF.7MHdhgSjnD3C3e5rEB6MEKxlqxaR4dnIgAi8)
```

Comment:
Prefix this name with `gke-` so the gke extension can automatically select it based on the release target name.